test/WebSites/ActivatorWebSite/TagHelpers/FooterTagHelper.cs

```diff
@@ -2,23 +2,28 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNet.Mvc;
-using Microsoft.AspNet.Mvc.Rendering;
 using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using Microsoft.Framework.WebEncoders;
 
 namespace ActivatorWebSite.TagHelpers
 {
     [TargetElement("body")]
     public class FooterTagHelper : TagHelper
     {
-        [Activate]
-        public IHtmlHelper HtmlHelper { get; set; }
+        public FooterTagHelper(IHtmlEncoder encoder)
```

Comment:
Well, the other option is to constructor inject an `HtmlHelper` and call `Contextualize()` on it after the `ViewContext` is set (or in the setter). Might be nice to show it's possible (regardless of the efficacy of using an `HtmlHelper` when you only need an `IHtmlEncoder`).

Further, `HtmlHelper.Encode()` doesn't use the `ViewContext`. So `Contextualize()` is only nice-to-have for this (fanciful) use case.